pithos/SearchDialog.py

```diff
@@ -15,52 +15,59 @@
 ### END LICENSE
 
 import html
-from gi.repository import GObject, Gtk
+from gi.repository import GObject, Gtk, Gdk
 
 from .gi_composites import GtkTemplate
 
 @GtkTemplate(ui='/io/github/Pithos/ui/SearchDialog.ui')
 class SearchDialog(Gtk.Dialog):
     __gtype_name__ = "SearchDialog"
+    __gsignals__ = {
+        "add-station": (GObject.SignalFlags.RUN_FIRST, None, (str,)),
+    }
 
     entry = GtkTemplate.Child()
-    treeview = GtkTemplate.Child()
+    search_results_model = GtkTemplate.Child()
 
-    def __init__(self, *args, **kwargs):
-        self.worker_run = kwargs["worker"]
-        del kwargs["worker"]
-
-        super().__init__(*args, use_header_bar=1, **kwargs)
+    def __init__(self, worker_run):
+        super().__init__()
```

Comment:
Always pass `**kwargs` to init, pygobject uses it for setting constructor properties.